lib/pronto/config_file.rb

```diff
@@ -35,8 +36,10 @@ class ConfigFile
       'format' => DEFAULT_MESSAGE_FORMAT
     }.freeze
 
-    def initialize(path = '.pronto.yml')
-      @path = path
+    def initialize
+      @path = ENV.fetch('PRONTO_CONFIG', DEFAULT_FILE_PATH)
+      return if @path == DEFAULT_FILE_PATH || File.exist?(@path)
```

Comment:
Since it will always be set it would mean that `.pronto.yml` would always be required if no custom file path is set. That means that every project should have such a config file.

It should only raise an error if a `--config` or `PRONTO_CONFIG` is used. In case it is not and pronto falls back to `.pronto.yml` (which is the default behavior) it should not raise an error if that file does not exist.